frontend/packages/metal3-plugin/src/constants.ts

```diff
@@ -1,6 +1,7 @@
 export const HOST_STATUS_READY = 'ready';
 export const HOST_STATUS_DISCOVERED = 'discovered';
 export const HOST_STATUS_OK = 'OK';
+export const HOST_STATUS_ERROR = 'error';
```

Comment:
This should be needed unless this is a value for `operationalStatus`. If yes, then it should be also added to `HOST_ERROR_STATES` array